website/docs/docs/deploy/job-settings.md

```diff
@@ -56,3 +66,33 @@ You can create a job and configure it to run on [scheduled days and times](/docs
 <Lightbox src ="/img/docs/dbt-cloud/using-dbt-cloud/triggers.jpg" width="85%" title="Configuring your job triggers"/>
 
 7. Select **Save**, then click **Run Now** to run your job. Click the run and watch its progress under **Run history**.
+
+</TabItem>
+
+<TabItem value="beta" label="Beta version">
+
+1. On your deployment environment page, click **Create Job** > **Deploy Job** to create a new deploy job. 
```

Comment:
hmm, there are 2 ways to create a deploy job so does it make sense to have environment page be our step 1?

Reply:
I don't think we should create environment as a first step. An environment though should be a pre-req. You have 1 environment for production, with lots of jobs in it.